### PR TITLE
fix: shebangs

### PR DESCRIPTION
**Pull request by lucasew**

### CODE REVIEW

**Comment by alextran1502** [comment]
Hello, what is the reason for this PR?

**Comment by lucasew** [comment]
> Hello, what is the reason for this PR?

Fixing the shebangs for non FHS compliant distros, I am working on another PR rn so it may change a bit but I sent this as a low hanging fruit.

**Comment by jrasm91** [comment]
It is still unclear to me why this is necessary and what value it brings. Is there a specific problem this is solving?

**Comment by lucasew** [comment]
> It is still unclear to me why this is necessary and what value it brings. Is there a specific problem this is solving?

Some distros like NixOS doesn't have anything in /bin. Termux is another example but because of the deployment complexity of this I think it's very unlikely this project will ever run there.

It's kind of a meme that NixOS users are the ones who fix shebangs tho xD but in more traditional distros will work the same way.

BTW there is also some scripts that don't have a shebang and the thing that sets up the executables doesn't work properly and that would need to be patched anyway.

**Review by alextran1502 — APPROVED** [review]

**Comment by alextran1502** [comment]
It seems like changing the shebang causes issue with running one of the test script

File: cli/src/api/open-api/git_push.sh [hunk 1]
```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env sh
```
Comment (jrasm91) [inline cli/src/api/open-api/git_push.sh:1]: This file is auto generated and doesn't need to be fixed.

File: mobile/openapi/git_push.sh [hunk 2]
```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env sh
```
Comment (jrasm91) [inline mobile/openapi/git_push.sh:1]: This file is also generated

File: web/src/api/open-api/git_push.sh [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env sh
```
Comment (jrasm91) [inline web/src/api/open-api/git_push.sh:1]: And this one.

File: mobile/openapi/git_push.sh (hunk 2 above)
Comment (lucasew) [reply mobile/openapi/git_push.sh]: Done

File: web/src/api/open-api/git_push.sh (hunk 3 above)
Comment (lucasew) [reply web/src/api/open-api/git_push.sh]: Done

**Pull request merged**